### PR TITLE
fix:  use data-title to add title text to oui-back-button

### PR DESCRIPTION
**Pull request by radireddy**
### Requirements

* Filling out the template is required. Any pull request that does not include enough information to be reviewed in a timely manner may be closed at the maintainers' discretion.
* All new code requires tests to ensure against regressions

## Title of the Pull Requests 
Added title text to oui-back-button using data-title attribute

### Description of the Change
In some places, data-title was not used to add a title for the back button. Instead, a span was used with custom style to make it look like back button title. I have added a title using data-title attribute in all these places.

### CODE REVIEW

File: client/app/dbaas/logs/detail/inputs/console/logs-inputs-console.html [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <div class="cui-page__content inputs-logs">
-    <oui-back-button class="back_button_inline_text"></oui-back-button>
-    <span class="oui-header_3" data-translate="inputs_logs_title" data-translate-values="{inputTitle: ctrl.input.data.info.title}"></span>
+    <oui-back-button></oui-back-button>
```
Comment (whispyy) [inline client/app/dbaas/logs/detail/inputs/console/logs-inputs-console.html]: The usage of oui-back-button allows to put a data-title inside like that :
`<oui-back-button data-title="{{ 'inputs_logs_title' | translate: {inputTitle: ctrl.input.data.info.title} }}"></oui-back-button>`

Comment (radireddy) [reply client/app/dbaas/logs/detail/inputs/console/logs-inputs-console.html]: I tried it and it does not work. I mentioned it in UI-Kit HipChat room but I did not get any response on it. Seems like a bug on oui-back-button component.

File: client/app/dbaas/logs/detail/inputs/console/logs-inputs-console.html [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
+    <span class="oui-header_3">{{ 'inputs_logs_title' | translate: {inputTitle: ctrl.input.data.info.title} }}</span>
```
[the remark this answers is not in the document]
Comment (radireddy) [reply client/app/dbaas/logs/detail/inputs/console/logs-inputs-console.html]: there is a bug with oui-back-button, it does not show translated text with parameters. This is a workaround till the issue is fixed in the component.

File: client/app/dbaas/logs/detail/roles/edit-permissions/edit-permissions.html [hunk 3]
```diff
@@ -1,6 +1,5 @@
 <div class="cui-page__content">
-    <oui-back-button class="back_button_inline_text"></oui-back-button>
-    <span class="oui-header_3" data-translate="logs_roles_edit_permissions_title"></span>
+    <oui-back-button data-title="{{ 'logs_roles_edit_permissions_title' | translate }}"></oui-back-button>
```
[the remark this answers is not in the document]
Comment (antleblanc) [reply client/app/dbaas/logs/detail/roles/edit-permissions/edit-permissions.html]: FYI upcoming changes: ovh-ux/ovh-ui-angular#205

**Comment by antleblanc** [comment]
@radireddy Could you please update/set labels?

File: client/app/dbaas/logs/detail/inputs/console/logs-inputs-console.html (hunk 1 above)
Comment (FredericEspiau) [reply client/app/dbaas/logs/detail/inputs/console/logs-inputs-console.html]: @AxelPeter have you heard about this ?

Comment (radireddy) [reply client/app/dbaas/logs/detail/inputs/console/logs-inputs-console.html]: created a jira task UK-44 to track this issue

File: client/app/dbaas/logs/detail/inputs/console/logs-inputs-console.html (hunk 2 above)
[the remark this answers is not in the document]
Comment (radireddy) [reply client/app/dbaas/logs/detail/inputs/console/logs-inputs-console.html]: created a jira task UK-44 to track this issue

**Comment by antleblanc** [comment]
@radireddy Existing a ticket related to this pull request? (If yes, could you please update the description with the ticket ID? thanks)

**Comment by jleveugle** [comment]
Hello @radireddy,
Your pull request is now closed due to inactivity.
Feel free to re-open it if you have any update.
Anna Bot for OVH UX <3

**Pull request closed without merging**